examples/pytorch/AxHyperOptimizationPTL/classifier.py

```diff
@@ -0,0 +1,165 @@
+import mlflow.pytorch
+import pytorch_lightning as pl
+import torch
+import torch.nn as nn
+import torchvision
+from mlflow.utils.autologging_utils import try_mlflow_log
+from sklearn.metrics import accuracy_score
+from torch.nn import functional as F
+from torch.utils.data import DataLoader
+from torch.utils.data.dataset import random_split
+from torchvision import transforms
+from pytorch_lightning.metrics.functional import accuracy
+
+
+class DataModule(pl.LightningDataModule):
+    def __init__(self):
+        super().__init__()
+        self.train_transforms = transforms.Compose(
+            [
+                transforms.RandomCrop(32, padding=4),
+                transforms.RandomHorizontalFlip(),
+                transforms.ToTensor(),
+                transforms.Normalize((0.4914, 0.4822, 0.4465), (0.2023, 0.1994, 0.2010)),
+            ]
+        )
+        self.test_transforms = transform_test = transforms.Compose(
+            [
+                transforms.ToTensor(),
+                transforms.Normalize((0.4914, 0.4822, 0.4465), (0.2023, 0.1994, 0.2010)),
+            ]
+        )
+
+    def prepare_data(self):
+        trainset = torchvision.datasets.CIFAR10(
```

Comment:
Similar to my comment on the iterative-pruning RP (https://github.com/mlflow/mlflow/pull/3884/files#r570644315), but do we really need to train LeNet with CIFAR10 to demostrate how Ax and MLflow work together? Can we just train a toy model with toy data?

Reply:
@harupy I am working on updating the code with Iris example.